lib/jekyll/converter.rb

```diff
@@ -8,7 +8,9 @@ class Converter < Plugin
     #
     # Returns the String prefix.
     def self.highlighter_prefix(highlighter_prefix = nil)
-      @highlighter_prefix = highlighter_prefix if highlighter_prefix
+      if !defined?(@highlighter_prefix) || !highlighter_prefix.nil?
```

Comment:
The description of the method says:

> When an argument is specified, the prefix will be set. If no argument is specified, the current prefix will be returned.

That's why i check for `nil` (the default argument value) explicitly instead of just a `truthy` value. It seemed more accurate according to the comment to me.
I added the `!defined?` check to set the `@higlighter_prefix` in this specific case even if the argument is `nil` to remove the warning and initialize the instance variable with `nil`.